packages/crew_sudo.rb

```diff
@@ -6,9 +8,8 @@ class Crew_sudo < Package
   version '1.1'
   license 'GPL-3'
   compatibility 'all'
-
-  source_url 'https://github.com/chromebrew/crew-sudo.git'
-  git_hashtag "v#{@version}"
+  source_url 'https://github.com/chromebrew/crew-sudo/archive/refs/tags/v1.1.tar.gz'
+  source_sha256 'f71d6a24bc6d1271a7ebbe255a53d08f55fd95d2430d9c14b8d10f82f29c610f'
```

Comment:
I also found out there is no crew_sudo.filelist in the manifest directory.  Can we add that too?  Probably the cleanest way to add is at https://github.com/chromebrew/crew-sudo along with a new release.

Reply:
Actually, you can probably just remove the updates from crew_sudo.rb here and just make them in #10246 instead.

Reply:
Crew_sudo doesn't need binaries.